flutter/ios/Classes/SwiftSentryFlutterPlugin.swift

```diff
@@ -1,29 +1,195 @@
+// swiftlint:disable all
```

Comment:
why? :)

Reply:
The purpose of linting is to improve the quality of the code. Just disabling all the warnings doesn't make sense to me. Picking out a few to disable is reasonable. For sure the formatting issues made sense to me. We can argue if we want to disable force casting in this file for example because it could be that you have a hard time avoiding this.

Reply:
✅